### ISSUE TITLE
Android Auto feature is not compliant

### ISSUE THREAD

**Issue by bmarty** [element-hq/element-android#4205]
Android Auto feature has been added in 1.3.2, but The PlayStore has rejected the upgrade with this reason:

> The app reads out the outgoing messages, the app should only read out the incoming messages

1.3.3 will disable Android Auto support, to unblock release 1.3.2, in the mean time let's investigate the problem

Related: #4078

**Comment by bbigras** [comment]
Can this be closed since #4247 is done?

**Comment by faan11** [comment]
It does not work. I'm using the last version of Element available in F-Droid.
The app is not present in the list.

**Comment by pucilpet** [comment]
@faan11 From another issue: "It seems like apps require some sort of entitlement granted by the Play Store to actually work on the Android Auto app or an Android Auto head unit. Debug builds will only work in the [Desktop Head Unit](https://developer.android.com/training/cars/testing#test-auto). I assume this also means the F-Droid build will not work with Android Auto."

I also had f-droid version earlier and it was not working. After reverting to google play store version, it is now showing up.
